src/lib/data/alignment.js

```diff
@@ -790,6 +800,112 @@ export default class Alignment {
     return true
   }
 
+  /**
+   * Adds a line break after the token
+   * @param {Token} token
+   * @returns {Boolean}
+   */
+  addLineBreakAfterToken (token) {

```

Comment:
it just occurred to me, we're going to need removeLineBreakAfter Token too....

Reply:
I think that it is good, that some new requirements are always borned with each my PR :)
I hope it means that my progress make some inspiration, Bridget :)

Reply:
added